source/more-info/backup-emergency-kit.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: "Backup emergency kit"
+description: "Information about backup emergency kit"
+---
+
+The backup emergency kit contains information needed to recover your backup, such as the encryption key.
```

Comment:
Not “such as”. It is the encryption key.

Reply:
It was changed very recently whatsoever, https://github.com/home-assistant/frontend/commit/637fe37ef4c46faf34dfa8693943a806c9758de1